frontend/src/scenes/exports/api.ts

```diff
@@ -132,6 +132,40 @@ export const useCreateExport = (): {
     return { createExport, ...state }
 }
 
+export const useDeleteExport = (
+    teamId: number,
+    exportId: string,
+    deleteCallback: () => Promise<void>
```

Comment:
Rather than passing in the callback here, could we instead use `deleteExport().then(...)`. This gives us e.g. `catch` handling as well e.g. to display a toast on error.

Reply:
Added toast and used `then(...)` 👍